### PR TITLE
Add github action to deploy storybook

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
     "lint:js": "eslint --ext .js,.svelte .",
     "lint:shell": "shellcheck all create",
     "storybook": "start-storybook -s ./public -p 6006",
+    "deploy-storybook": "storybook-to-ghpages",
```
Comment (robhudson) [inline package.json:13]: @openjck On 2nd thought, perhaps this makes more sense as part of the github actions installation step? Unless you think some devs would want to deploy manually, I can move it there.

File: package.json [hunk 2]
```diff
@@ -19,6 +20,7 @@
     "@babel/preset-env": "7.10.2",
     "@storybook/addon-knobs": "^5.3.18",
     "@storybook/addon-storysource": "^5.3.18",
+    "@storybook/storybook-deployer": "^2.8.6",
```
Comment (robhudson) [inline package.json]: @openjck On 2nd thought, perhaps this makes more sense as part of the github actions installation step? Unless you think some devs would want to deploy manually, I can move it there.

File: package.json (hunk 1 above)
Comment (robhudson) [reply package.json:13]: Oops, meant to comment on the dependency, not the run command. But this would go away too.

File: package.json (hunk 2 above)
Comment (openjck) [reply package.json]: That works for me. I don't expect we would deploy it manually very often if ever. The only other benefit I could imagine of listing it here is that we could use dependabot to keep it up-to-date, but if there's another way of accomplishing that, that's fine with me.

**Pull request merged**